jhub_apps/spawner/command.py

```diff
@@ -54,7 +54,7 @@ def get_substituted_args(self, **kwargs):
         TString("$python_exec"),
         "-m",
         "jhsingle_native_proxy.main",
-        TString("--authtype=$authtype"),
+        TString("--authtype=$authtype --no-force-alive"),
```

Comment:
I believe the failing tests should work if you put this in a separate line:

```python
 TString("--authtype=$authtype"),
"--no-force-alive",
```

Reply:
:man_facepalming: 
thanks!